app/views/external_users/claims/_main_claims_list.html.haml

```diff
@@ -5,90 +5,93 @@
         = your_claims_header
       %span.claim-count
         - if @claims.none?
-          = t("shared.no_claims_found")
+          = t('shared.no_claims_found')
         - else
-          = "Showing "
+          = 'Showing '
           #{pluralize(@claims.unscope(:order).size, 'claim')} of #{@claims.total_count}
```

Comment:
I'm not sure this will work in terms of string interpolation - `"#{needs to be in double quotes}"`

Reply:
It works without double quotes, however for consistency, I have update it to use double quotes.